cookbook-react/src/recipes/utils/validations/index.test.ts

```diff
@@ -0,0 +1,99 @@
+import * as Validator from '.';
```

Comment:
i dont really like `.` as import, maybe `./index.ts` ? but its just a comment not mandatory